Readme.mkd

```diff
@@ -72,6 +72,38 @@ $.ajax({
 });
 ```
 
+In case, if you need to find the rank of the object w.r.t to the ranked column, you could do this by accessing `{column_name}_rank` method available on the model objects, where `{column_name}` is your resource ranking column.
```

Comment:
I hope you don't mind but I've re-written this for you to remove the abbreviations and to make the meaning clearer:

```
If you need to find the rank of an item with respect to other ranked items, you can use the `{column_name}_rank` method on the model instance. `{column_name}` is your resource ranking column.

Following on from our examples above, the `row_order_rank` method will return the position of the duck object in the list with respect to the order defined by the row_order column.

Duck.rank(:row_order).first.row_order_rank # => 0
Duck.rank(:row_order).third.row_order_rank # => 2
```

> NOTE - The rank method would return 1 for the first duck in the list

With regards to the above, this appears to be incorrect as your tests say it would return 0 (which is better anyway) so I think that line can be removed as the code examples already indicate that to be the case :)

Reply:
Thanks 👍 

> 
>```ruby
>Duck.rank(:row_order).first.row_order_rank # => 0
>Duck.rank(:row_order).third.row_order_rank # => 2
>```

☝️ is definitely more helpful and a better way of specifying the usage.

Reply:
Also, thanks for pointing this out. It was always intended to be 0 😓 

> NOTE - The rank method would return 1 for the first duck in the list